README.md

```diff
@@ -1 +1,42 @@
-# microverseTrial2
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Microverse Trial 2
+
+> This project is a trial for admission at microverse 
+
+![screenshot](./app_screenshot.png)
+
+## Built With
+
+- HTML
+- CSS
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- Click the green Code button on the repo.
+- Copy the project url
+- In your local PC, open your terminal in the folder you would like to clone the project.
+- type "git clone (project-url)"
```

Comment:
Please use explicit commands. This makes it easier to get started with your project. You should have pointed this out in your review.

Reply:
I don't know what is explicit commands, What do you mean by it?